### DISCUSSION TITLE
How to build the development branch for ios?

### DISCUSSION THREAD

**Discussion by DennisFeldbusch** [ahrm/sioyek#936] [General]
How to build the development branch for ios?

**Comment by ahrm** [comment]
Tell me if you manage to do it :) .

**Reply by DennisFeldbusch** [reply]
I don't even have a clue how to start. I'm new to QT and it's cross-platform capabilities.

**Comment by DennisFeldbusch** [comment]
@ahrm in [778](https://github.com/ahrm/sioyek/issues/778) it seems like you have/ had a ios project set up. 
Would you like to share how you ported the qt project to ios?

**Reply by ahrm** [reply]
I did it a few months ago, I don't remember the details. I remember it being quite complicated though, I had to manually strip some symbols off the object files to get it working.

**Comment by DennisFeldbusch** [comment]
@ahrm did you manage to build the mupdf library for ios? And do you remember how?

**Reply by ahrm** [reply]
I remember using some build scripts from this old repo: https://github.com/ArtifexSoftware/mupdf-ios-viewer. Probably needs some modification though. I think I posted the diffs of the modifications I made to the `mupdf` discord channel (I was asking some questions there), You might be able to find them by searching Artifex discord channel for user `hexomancer`.
